eng/testing/tests.mobile.targets

```diff
@@ -82,12 +85,16 @@
       <AndroidAbi Condition="'$(TargetArchitecture)' == 'x86'">x86</AndroidAbi>
 
       <MainLibraryFileName Condition="'$(MainLibraryFileName)' == ''">AndroidTestRunner.dll</MainLibraryFileName>
+      <RuntimeConfigFilePath>$(PublishDir)$(AssemblyName).runtimeconfig.json</RuntimeConfigFilePath>
+      <OutputFilePath>$(PublishDir)output.pefile</OutputFilePath>
```

Comment:
1. property names are global - maybe it's better to use something less generic.
2. `output.pefile` is a bad name: (1) "output" doesn't really tell us anything about the file's content, (2) ".pefile" is a bad extension - it's not a PE file anymore.  Why not `runtimeconfig.bin` or something?

Reply:
Fixed. Thanks!